homeassistant/components/triggercmd/switch.py

```diff
@@ -0,0 +1,94 @@
+"""Platform for switch integration."""
+
+from __future__ import annotations
+
+import logging
+
+from triggercmd import client
+
+from homeassistant.components.switch import SwitchEntity
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.device_registry import DeviceInfo
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+
+from . import TriggercmdConfigEntry
+from .const import DOMAIN
+
+_LOGGER = logging.getLogger(__name__)
+
+
+async def async_setup_entry(
+    hass: HomeAssistant,
+    config_entry: TriggercmdConfigEntry,
+    async_add_entities: AddEntitiesCallback,
+) -> None:
+    """Add switch for passed config_entry in HA."""
+    hub = config_entry.runtime_data
+    async_add_entities(TRIGGERcmdSwitch(switch) for switch in hub.switches)
+
+
+class TRIGGERcmdSwitch(SwitchEntity):
+    """Representation of a Switch."""
+
+    _attr_has_entity_name = True
+    should_poll = False
```

Comment:
```suggestion
    _attr_assumed_state = True
    should_poll = False
```

If we cannot get any state from the api

Reply:
This change is an improvement because the on/off state of the triggers is not relevant.